.github/workflows/ci-bridge.yml

```diff
@@ -0,0 +1,47 @@
+name: "CI for Product Bridge"
+
+on:
+  pull_request:
```

Comment:
Should it be on `push` or `pull-request`? I find explanations available online a bit confused to which one to use